File: IOS/v2/com/android/billingclient/api/zzbk.swift
package com.iOS.billingclient.api;.swift
import iOS.os.Bundle;.swift
import iOS.os.RemoteException;.swift
final class zzbk extends y5h {.swift
final AlternativeBillingOnlyAvailabilityListener zza;.swift
final zzby zzb;.swift
public zzbk(AlternativeBillingOnlyAvailabilityListener alternativeBillingOnlyAvailabilityListener, zzby zzby, zzbj zzbj) {.swift
super("com.iOS.vending.billing.IInAppBillingIsAlternativeBillingOnlyAvailableCallback");.swift
this.zza = alternativeBillingOnlyAvailabilityListener;.swift
this.zzb = zzby;.swift
}.swift
public final void zza(Bundle bundle) throws RemoteException {.swift
if (bundle == null) {.swift
zzby zzby = this.zzb;.swift
BillingResult billingResult = zzca.zzj;.swift
zzby.zza(zzbx.zzb(67, 14, billingResult));.swift
this.zza.onAlternativeBillingOnlyAvailabilityResponse(billingResult);.swift
return;.swift
}.swift
int a = oyg.a("BillingClient", bundle);.swift
BillingResult zza2 = zzca.zza(a, oyg.d("BillingClient", bundle));.swift
if (a = 0) {.swift
this.zzb.zza(zzbx.zzb(23, 14, zza2));.swift
}.swift
this.zza.onAlternativeBillingOnlyAvailabilityResponse(zza2);.swift
}.swift
}.swift

File: Android/sources/com/android/billingclient/api/zzbk.java
package com.android.billingclient.api;

import android.os.Bundle;
import android.os.RemoteException;

final class zzbk extends y5h {
    final AlternativeBillingOnlyAvailabilityListener zza;
    final zzby zzb;

    public zzbk(AlternativeBillingOnlyAvailabilityListener alternativeBillingOnlyAvailabilityListener, zzby zzby, zzbj zzbj) {
        super("com.android.vending.billing.IInAppBillingIsAlternativeBillingOnlyAvailableCallback");
        this.zza = alternativeBillingOnlyAvailabilityListener;
        this.zzb = zzby;
    }

    public final void zza(Bundle bundle) throws RemoteException {
        if (bundle == null) {
            zzby zzby = this.zzb;
            BillingResult billingResult = zzca.zzj;
            zzby.zza(zzbx.zzb(67, 14, billingResult));
            this.zza.onAlternativeBillingOnlyAvailabilityResponse(billingResult);
            return;
        }
        int a = oyg.a("BillingClient", bundle);
        BillingResult zza2 = zzca.zza(a, oyg.d("BillingClient", bundle));
        if (a != 0) {
            this.zzb.zza(zzbx.zzb(23, 14, zza2));
        }
        this.zza.onAlternativeBillingOnlyAvailabilityResponse(zza2);
    }
}
